python/lsst/pipe/tasks/skyCorrection.py

```diff
@@ -387,6 +400,37 @@ def _restoreBackgroundRefineMask(self, calExps, calBkgs):
             skyCorrBases.append(skyCorrBase)
         return calExps, skyCorrBases
 
+    def _restoreVisitBackground(self, calExps, calBkgs):
+        """Restores the full focal-plane background to a visit.
+        Runs after _subtractSkyFrame() if doBgModel1=False.
+
+        Parameters
+        ----------
+        calExps : `list` [`lsst.afw.image.exposure.ExposureF`]
+            Calibrated exposures to be background subtracted.
+        calBkgs : `list` [`lsst.afw.math._backgroundList.BackgroundList`]
+            Background lists associated with the input calibrated exposures.
+
+        Returns
+        -------
+        calExps : `list` [`lsst.afw.image.maskedImage.MaskedImageF`]
```

Comment:
This worried me originally, as the input `calExps` are `ExposureF`, but the output is a `MaskedImageF`? However, I think this returns section is going away in any case (see other related comment). You can probably replace this section with a `Notes` stating that the inputs are modified in-place.